cms/djangoapps/contentstore/views/course.py

```diff
@@ -261,6 +262,9 @@ def course_handler(request, course_key_string=None):
             if course_key_string is None:
                 return redirect(reverse("home"))
             else:
+                # Revoke sudo privileges from a request explicitly
```

Comment:
Is the goal to revoke sudo privileges any time a user navigates away from a `sudo_required` page? If so, this particular implementation seems like an incomplete solution to me, as we will never know where a course author/administrator will go after viewing such a page. Isn't that what the `SUDO_COOKIE_AGE` setting is for?